docs/user/cli/geotools.rst

```diff
@@ -82,8 +82,8 @@ folder. To install JARs, use the script provided with the distribution:
     $ bin/install-hadoop.sh lib
 
 If you installed JARs for any additional data stores, you will need to add them to the Hadoop libjars path
-by modifying the file ``org/locationtech/geomesa/geotools/tools/ingest-libjars.list`` inside the
-``lib/geomesa-gt-tools_2.11-$VERSION.jar`` JAR.
+by modifying the file ``org/locationtech/geomesa/geotools/tools/gt-libjars.list`` inside the JAR
```

Comment:
(Idle question) Why did this filename change?

Reply:
I separated out the base jars used by ingest into the tools jar, then have a back-end-specific file for each distro (as noted in the commit message :).

Reply:
Oh man, I read +7k lines and -5k lines and I miss the commit message.